_std/defines/component_defines.dm

```diff
@@ -325,6 +330,14 @@
 
 #define COMSIG_SUSSY_PHRASE "sussy"
 
+// ---- Transfer system ----
+/// When an item is requested to be transfered to the output target (/obj/item/)
+#define COMSIG_INCOMING_TRANSFER "incoming_tx"
+/// When the target wants to send an item to an output (/obj/item/)
+#define COMSIG_OUTGOING_TRANSFER "outgoing_tx"
+/// Return whether the target should allow receiving items from the given atom (/atom)
+#define COMSIG_CAN_LINK "permit_tx"
```

Comment:
I feel like putting the common part first is more in line with current naming convention. I.e. `COMSIG_TRANSFER_INCOMING`, `COMSIG_TRANSFER_OUTGOING`, `COMSIG_TRANSFER_CAN_LINK`.